AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "galleryItemUri",
+            "in": "body",
+            "description": "Gallery Item Uri from where the gallery package is downloaded.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GalleryItemUriPayload"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.gallery.admin/galleryItems/{galleryItemName}": {
+      "get": {
+        "tags": [
+          "AdminGalleryItems"
+        ],
+        "description": "Gets a specific gallery registration.",
+        "operationId": "AdminGalleryItems_Get",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "galleryItemName",
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "AdminGalleryItems"
+        ],
+        "description": "Deletes a specific gallery registration.",
+        "operationId": "AdminGalleryItems_Delete",
+        "consumes": [],
+        "produces": [],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "galleryItemName",
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "200": {
+            "description": "OK"
```

Comment:
200, does not have a schema?
Can you please explain in the description under what circumstances will the server return a 200 v/s a 204? This will enhance the user experience.